cdap-docs/admin-manual/source/security.rst

```diff
@@ -450,16 +449,17 @@ will use ``https`` instead of ``http``.
 
     {"auth_uri":["http://localhost:10009/token"]}
 
-- Submit a username and password to one of the URIs to obtain an ``AccessToken``::
+- Submit a username and password to one of the authentication URIs (``<auth-uri>``) to
+  obtain an ``AccessToken``::
 
-    GET -u <username>:<password> <auth-uri>
+    GET <auth-uri> <username>:<password>
```

Comment:
I think the problem is that we're trying to simplify the documentation with regard to authentication. It's much easier to `-u username:password` than to explain the encoded authorization header those options create. Personally, I think that we should just describe the header and how it's constructed, since we only need it in this one place.

https://en.wikipedia.org/wiki/Basic_access_authentication#Client_side

After all, we describe the header itself for our token. We should do the same for fetching the token in the first place.